src/test/java/com/openlattice/datastore/test/HazelcastTestHarness.java

```diff
@@ -20,62 +20,60 @@
 
 package com.openlattice.datastore.test;
 
-import java.util.Map.Entry;
-
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
-import org.junit.Test;
-
 import com.hazelcast.config.Config;
-import com.hazelcast.config.GroupConfig;
 import com.hazelcast.config.JoinConfig;
 import com.hazelcast.config.MulticastConfig;
 import com.hazelcast.config.NetworkConfig;
 import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.IMap;
-import com.hazelcast.map.AbstractEntryProcessor;
+import com.hazelcast.map.EntryProcessor;
+import com.hazelcast.map.IMap;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Map.Entry;
 
 public class HazelcastTestHarness {
```

Comment:
We should be able to just delete this file and anything like this should a best a util in test.